src/utils.js

```diff
@@ -36,7 +36,33 @@ export function patchArticleVote(article_id, votes) {
     inc_votes: votes,
   };
 
-  return ncNewsApi.patch(`/articles/${article_id}`, patchVotes).then((data) => {
-    return data;
-  });
+  return ncNewsApi
+    .patch(`/articles/${article_id}`, patchVotes)
+    .then((data) => {
+      return data;
+    })
+    .catch((err) => console.log(err));
+}
+
+export function fetchUsers() {
+  return ncNewsApi
+    .get('/users')
+    .then((res) => {
+      return res.data;
+    })
+    .catch((err) => console.log(err));
+}
+
+export function postComment( username, newCommentText, article_id ) {
```

Comment:
Nice work, could this file be renamed to api to reflect what the functions inside of it are for. utils may be deemed more generic.